python/idsse_common/idsse/common/rabbitmq_utils.py

```diff
@@ -89,7 +90,7 @@ def _initialize_exchange_and_queue(
 
     # Do not try to declare the default exchange. It already exists
     if exch.name != '':
-        channel.exchange_declare(exchange=exch.name, exchange_type=exch.type)
+        channel.exchange_declare(exchange=exch.name, exchange_type=exch.type, durable=exch.durable)
```

Comment:
Are there other args that we should be including, ie autoDelete and durable?

Reply:
They haven't been necessary until now so probably not.